notification_service_tests.sh

```diff
@@ -69,8 +77,29 @@ function init_db() {
     rm config.toml
 }
 
+# mechanism to chain more trap commands added in different parts of this script
+exit_trap_command=""
+function cleanup {
+    eval "$exit_trap_command"
+}
+trap cleanup EXIT
+
+function add_exit_trap {
+    local to_add=$1
+    if [[ -z "$exit_trap_command" ]]
+    then
+        exit_trap_command="$to_add"
+    else
+        exit_trap_command="$exit_trap_command; $to_add"
+    fi
+}
+
 [ "$NOVENV" != "1" ] && install_reqs || prepare_venv || exit 1
 
+# install mock dependencies and launch mocked token refreshment server (need even without other mocked services)
+python3 $(which pip3) install -r requirements/mocks.txt
+start_mocked_token_refreshment_server
```

Comment:
why is it outside `start_mocked_dependencies`?

Reply:
Well, this is something, that cannot be deployed locally. But it's true, that real service log will not work without real token, so deployment of the mocked token server is meaningless. I will put it back :)

Reply:
Should be done. I also changed the response of service-log mock to delete logs, it did not return the response code correctly.